xc/common/utils/prjxray_arch_import.py

```diff
@@ -521,6 +521,8 @@ def is_in_roi(conn, roi, tile_pkey):
     "IBUFDS_GTE2": ("Y", 2),
     "IPAD": (None, None),
     "OPAD": (None, None),
+    "DSP48E1": (None, None),
+    "TIEOFF": (None, None),
```

Comment:
The `TIEOFF` pin is actually not really required as it contains hooks to the power rails (GND and VCC), these should already be included in the RR graph.

Reply:
But if I don't add it,  it will report an error